app/interactors/tasks/destroy.rb

```diff
@@ -2,10 +2,14 @@ module Tasks
   class Destroy
     include Interactor
 
-    delegate :task, to: :context
+    delegate :task, :project, :current_user, to: :context
```

Comment:
Same like in https://github.com/fs/task_tracker_itis_2023/pull/100/files#r1385412493